src/components/PageHeader/PageHeader.styled.tsx

```diff
@@ -1,8 +1,8 @@
 import theme from "@root/theme";
 import styled from "styled-components";
 
-export const StyledDrawerHeader = styled.div`
-	background-color: ${theme.newColors.grey2["100"]};
+export const StyledPageHeader = styled.div`
```

Comment:
Component is missing bottom border (should be the same as Form, DataView, and SummaryPageTopComponent)